GitInfo.txt

```diff
@@ -1 +1 @@
-4.1.0
\ No newline at end of file
+4.2.0
```

Comment:
```suggestion
4.1.0
```

Reply:
@PureWeen should i commit it?

Reply:
I'd say yes

that's not something I see should be changing with your PR

Reply:
rebased